src/seedu/addressbook/storage/StorageFile.java

```diff
@@ -88,6 +88,9 @@ public void save(AddressBook addressBook) throws StorageOperationException {
         /* Note: Note the 'try with resource' statement below.
          * More info: https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html
          */
+    	if(!path.toFile().exists()){
+    		System.out.println("You do not have the storage file.");
```

Comment:
Indentation problem here?. Our coding standard requires you to use 4 spaces instead of tabs. You can configure Eclipse to convert tabs to spaces automatically.